Orange/data/table.py

```diff
@@ -970,6 +1001,7 @@ def ensure_copy(self):
         """
         Ensure that the table owns its data; copy arrays when necessary.
         """
+
```

Comment:
If you are linting unrelated parts of the code, use a separate commit, please. I would even prefer if you do not lint code you were not changing because then the history is easier to understand.

I think, of all Orange source code, I have been looking into the history of the Table the most. The clearer it is, the better.